### PR TITLE
data: Eric-Obeng

### PR DESCRIPTION
**Pull request by Eric-Obeng**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6566"></a>

### CODE REVIEW

**Comment by Eric-Obeng** [comment]
My linkfree profile has been edited successfully

File: data/Eric-Obeng.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Eric Obeng",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A Frontend Developer passionate about learning new technologies to help solve problems",
+  "avatar": "https://github.com/Eric-Obeng.png",
```
Comment (Sindhuinti) [inline data/Eric-Obeng.json:6]: Hey!! No need to add avatar here. It will be auto imported from your github profile.

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

**Comment by Eric-Obeng** [comment]
Okay Thank you.

**Review by Eric-Obeng — COMMENTED** [review]
Avatar removed

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/Eric-Obeng.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Eric-Obeng",
+  "displayStatsPublic": true,
```
[the remark this answers is not in the document]
Comment (Eric-Obeng) [reply data/Eric-Obeng.json]: displayStatusPublic removed

**Comment by kumarsonsoff3** [comment]
Duplicate of #6567

**Comment by kumarsonsoff3** [comment]
I'll be closing this PR since it seems to be a duplicate of PR #6567. The one with the latest and updated changes has already been merged!!

**Comment by Eric-Obeng** [comment]
Okay Sir
Thank you

**Pull request closed without merging**